log.go

```diff
@@ -3,33 +3,44 @@ package turnpike
 import (
 	"errors"
 	"fmt"
-	glog "log"
+	"log"
 	"os"
 )
 
 var (
-	logFlags = glog.Ldate | glog.Ltime | glog.Lshortfile
-	log      *glog.Logger
+	loggerFlags = log.Ldate | log.Ltime | log.Lshortfile
+	logger      Logger
```

Comment:
Yeah, I've noticed that before too.  It's an annoying behavior (basically have to revert that hunk after each save) but in this case I think it is worth it.